collector/indices.go

```diff
@@ -93,6 +93,270 @@ func NewIndices(logger log.Logger, client *http.Client, url *url.URL) *Indices {
 				},
 				Labels: defaultIndexLabelValues,
 			},
+			{
+				Type: prometheus.GaugeValue,
+				Desc: prometheus.NewDesc(
+					prometheus.BuildFQName(namespace, "indices", "segment_count_primary"),
+					"Current number of segements with only primary shards on all nodes",
```

Comment:
` s/segements/segments/g`  (not only here)

Reply:
Thanks for catching! Will update the PR after lunch.